app/src/main/java/com/buzzardparking/buzzard/activities/MapActivity.java

```diff
@@ -78,9 +80,10 @@
     private static final int SETTING_REQUEST_CODE = 2;
     private static final int OVERLAY_REQUEST_CODE = 99;
 
-    // Singleton instance of map
+    // Singleton instance for permissions
```

Comment:
These instances are maintained to for making sure things are loaded when moving between states.